.buildkite/pipeline.yml

```diff
@@ -191,3 +183,74 @@ steps:
   #
   - label: 'Conditionally trigger full set of tests'
     command: sh -c .buildkite/pipeline_trigger.sh
+
+  - block: "Build Windows fixtures"
+    key: 'block-windows'
+    prompt: "Build Windows fixtures - Is a suitable agent running?"
+
+  #
+  # TODO Once Windows infrastructure is fully in place, refactor the following steps into the basic and full pipelines.
+  #
+  - label: Build Unity 2017 Windows test fixture
+    key: 'windows-2017-fixture'
+    depends_on: 'block-windows'
+    timeout_in_minutes: 30
+    agents:
+      queue: opensource-windows-unity
+    env:
+      UNITY_VERSION: "2017.4.40f1"
+    command:
+      # Using the artifacts plugin v1.3 on Windows seem to break the whole step
+      - 'buildkite-agent artifact download "Bugsnag.unitypackage" .'
+      - 'cd test\\desktop'
+      - 'c:\\Progra~1\\Git\\bin\\bash.exe ./features/scripts/build_maze_runner.sh'
```

Comment:
I expect the answer is no, but is there no way we can either avoid using both Win and Linux paths in a single line?

Likewise, is it possible to not need the escape slashes in the windows paths?

Reply:
I think the escapes on `\` are needed because it's a YAML file, but now I think about it I think you can actually use forward slashes in Windows paths anyway... trialling that now.